jerry-core/jmem/jmem-heap.c

```diff
@@ -181,7 +183,7 @@ jmem_heap_finalize (void)
 {
   JERRY_ASSERT (JERRY_CONTEXT (jmem_heap_allocated_size) == 0);
 #ifndef JERRY_SYSTEM_ALLOCATOR
-  VALGRIND_NOACCESS_SPACE (&JERRY_HEAP_CONTEXT (first), sizeof (jmem_heap_t));
+  VALGRIND_NOACCESS_SPACE (&JERRY_HEAP_CONTEXT (first), JMEM_HEAP_SIZE);
```

Comment:
My concern 2:
Can I use `JMEM_HEAP_SIZE` to replace the `sizeof (jmem_heap_t)`, are they always same?